cartridges/bm_algolia/cartridge/controllers/AlgoliaBM.js

```diff
@@ -43,6 +43,7 @@ function handleSettings() {
         var algoliaEnableContentSearch = ('EnableContentSearch' in params) && (params.EnableContentSearch.submitted === true);
         var algoliaEnableRecommend = ('EnableRecommend' in params) && (params.EnableRecommend.submitted === true);
         var algoliaEnablePricingLazyLoad = ('EnablePricingLazyLoad' in params) && (params.EnablePricingLazyLoad.submitted === true);
+        var algoliaEnableRealTimeInventory = ('EnableRealTimeInventory' in params) && (params.EnableRealTimeInventory.submitted === true);
```

Comment:
Naming suggestion (open for discussion): shouln't we put "Hook" somewhere in the name so it's clear that it enables a hook?
Such as `EnableRealTimeInventoryHook`?

Reply:
yes it makes sense. On the next [PR](https://github.com/algolia/algoliasearch-sfcc-b2c/pull/224), I will update it.